express-post-json/index.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const grades = {};
+let nextId = 1;
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  res.json(grades);
+});
```

Comment:
Ok, now this needs to get updated so that it responds with an array, like in `express-get-json`.